tools/webpack-jumpstart

```diff
@@ -44,18 +47,29 @@ if [ "${NODE_ENV-}" = "development" ]; then
     exit 1
 fi
 
-if ! git diff --quiet -- ':^test' ':^packit.yaml' ':^.github'; then
-    echo 'jumpstart: uncommitted local changes, skipping download' >&2
-    exit 1
```

Comment:
I really don't like dropping this check unconditionally. Now downloading the cache would claim to work even with local modifications, but then do that in vain and rebuild stuff anyway. To me as a developer this is too mushy and unpredictable -- either the cache applies to my tree or it doesn't. In the latter case, rebuilding webpacks is not really the end of the world.

Reply:
See the above comment about my alternate proposal.  We could maybe include this behaviour in `--best`: having some dirty working tree is kinda similar to having some commits on top of the most recent cache object?